src/utils/tokens.ts

```diff
@@ -360,8 +360,9 @@ const tokens = {
     }
 
     if (foundAmount < depositAmount) {
+      const availableAmount = selectedUtxos.available;
       throw new InsufficientFundsError(
-        `Not enough HTR tokens for deposit: ${depositAmount} required, ${foundAmount} available`
+        `Not enough HTR tokens for deposit: ${depositAmount} required, ${availableAmount} available`
```

Comment:
The error message is used as an error feedback message for users on Wallet Desktop. We could have removed the "available" part from the message as an easier solution, however as it is used in the frontend Carneiro and I discussed it would be better to change the "internals" rather then the message.